packages/flutter/test/material/app_test.dart

```diff
@@ -381,6 +381,23 @@ void main() {
     expect(log, <String>['onGenerateRoute /', 'onUnknownRoute /']);
   });
 
+  testWidgets('custom navigator with without routes supported', (WidgetTester tester) async {
```

Comment:
Nit: with or without?

Reply:
D'oh. Changed my mind mid stream and my hands didn't get the memo. Thanks for catching it.